src/mpid/ch4/netmod/ofi/ofi_rma.c

```diff
@@ -47,6 +47,13 @@ void MPIDI_OFI_complete_chunks(MPIDI_OFI_win_request_t * winreq)
     }
 
     winreq->chunks = NULL;
+    if (winreq->rma_type == MPIDI_OFI_PUT) {
+        MPIR_Datatype_release_if_not_builtin(winreq->noncontig.put.origin.datatype);
+        MPIR_Datatype_release_if_not_builtin(winreq->noncontig.put.target.datatype);
+    } else {
```

Comment:
```suggestion
    } else if (winreq->rma_type == MPIDI_OFI_GET) {
```

Reply:
Besides MPIDI_OFI_PUT and MPIDI_OFI_GET, will rma operations with other `rma_type` call this function?

Reply:
Not right now, but they might in the future.

Reply:
alright, I can change it.